SuperChamps/contracts/Ecosystem/SCVirtualAssetEvents.sol

```diff
@@ -31,6 +33,14 @@ contract SCVirtualAssetEvents {
     function EmitGameUserActivity(string memory user_id, string memory data) external isPermissionedUser {
         emit GameUserActivity(user_id, data);
     }
+
+    function EmitGameUserAssetWithSign(string memory user_id, string memory asset_id, int256 delta, string memory data, string memory sign) external {
+        emit GameUserAssetWithSign(user_id, asset_id, delta, data, sign);
+    }
+
+    function EmitGameUserActivityWithSign(string memory user_id, string memory data, string memory sign) external {
```

Comment:
Please use bytes instead of string for signature. Bytes is standard data type used for signature data in solidity.